transformations/mr_value_replacement/transformation.py

```diff
@@ -0,0 +1,88 @@
+import torchtext.vocab
+import torch
+from torchtext.data import get_tokenizer
+from interfaces.KeyValuePairsOperation import KeyValuePairsOperation
+import spacy
+from itertools import product
+import random
+from random import sample
+from tasks.TaskTypes import TaskType
+
+class MRValueReplacement(KeyValuePairsOperation):
+    tasks = [TaskType.E2E_TASK]
+    languages = ["en"]
+  
+    def __init__(
+        self, seed=0, n_similar=10, max_outputs=1):
+        super().__init__(seed, max_outputs=max_outputs)
+
+        random.seed(seed)
+        self.glove = torchtext.vocab.GloVe(name = "6B", dim = "100")
```

Comment:
Also seems Glove is also used in the "diverse_paraphrase" transformation. It might be better if there is a common Glove model just like a common spacy model?

Reply:
Sure, I think so. Can I change this? or you could do this?

Reply:
It is done.